src/include/rados/librados.hpp

```diff
@@ -14,6 +14,18 @@
 #include "librados_fwd.hpp"
 #include "rados_types.hpp"
 
+#define WITH_JAGER //will be removed after testing
+
+#ifdef WITH_JAGER
+  #include "../../common/tracer.h"
```

Comment:
these `include/rados/` headers are special, in that they're packaged in `librados-devel` so that other people can write programs with a rados client. src/common/tracer.h is in the ceph tree, so the packaged librados headers won't have access to it

maybe we should avoid any tracing in librados for now

Reply:
@cbodley noted

Reply:
@cbodley 
Should I just move my tracer.h and tracer.cc file to `/include/Jaeger_tracer` and then make a seperate cmake for that, will that work?